test-definitions/fern/exhaustive/definition/req-bodies.yml

```diff
@@ -0,0 +1,33 @@
+imports: 
```

Comment:
lets rename this to inlined-requests.yml

Reply:
was actually thinking of deleting this, these endpoints are now redundant due to additions in /endpoints/object.yml. is that ok?

Reply:
we actually need these because inlined requests touch a different part of the code than referenced requests